packages/components/src/systems/ScreeningAlerts/ScreeningAlerts.tsx

```diff
@@ -0,0 +1,27 @@
+import { ScreeningAlert, ScreeningAlertType } from './ScreeningAlert';
+
+/**
+ * This component represents multiple alerts for a given entity.
+ *
+ * The owningId property is used to demonstrate that this alert is attached to the provided ID
```

Comment:
provided draft id?

Reply:
theoretically it doesn't need to be the draft id - i did it this way in case we wanted to show alerting under other things as well (such as medical history or allergies) - "attaching" an alert to an entity means filtering out anything about it - OR, show the alert in its entirty and don't have an "owning id" to show everything.